graphql-api/src/main/java/com/expediagroup/streamplatform/streamregistry/graphql/model/GraphQLInfrastructure.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (C) 2018-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.streamplatform.streamregistry.graphql.model;
+
+import java.util.List;
+
+import lombok.Builder;
```

Comment:
dreaming of kotlin.... l-)

Reply:
Interesting you say that :) We were just talking about this today...